registry/coder-labs/modules/codex/main.tf

```diff
@@ -53,41 +53,21 @@ variable "codex_version" {
   default     = "" # empty string means the latest available version
 }
 
-variable "extra_codex_settings_toml" {
+variable "base_config_toml" {
```

Comment:
@DevelopmentCats, are these breaking changes? We are changing the input names?

Reply:
@35C4n0r what do you think? We can bump the major version, so I am fine with these. But we have to agree on the value it adds.